Task1/ClassLibrary/StartUp.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Microsoft.Extensions.Configuration;
+using System.IO;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace LoggerClassLibrary
+{
+    class StartUp
+    {
+        public Dictionary<string, List<string>> DictSettings = new Dictionary<string, List<string>>()
```

Comment:
Make this property as `private`. We should avoid using public fields from one file to another (encapsulation).